.changeset/fresh-frogs-heal.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firestore": patch
+"@firebase/webchannel-wrapper": patch
+---
+
+Fix an [issue](https://github.com/firebase/firebase-js-sdk/issues/7581) where Firestore was incorrectly using XHR instead of fetch for streaming response.
```

Comment:
nit: Remove the link to the issue; since your PR description explicitly specifies "Fixes: #7581" the link will be automatically added to the release notes.

For example, see how https://firebase.google.com/support/release-notes/js#version_1030_-_august_22_2023 was generated from https://github.com/firebase/firebase-js-sdk/pull/7569

Reply:
Done.